client/jetpack-connect/user-type/form.jsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import '../style.scss';
+
+class UserTypeForm extends Component {
+	static propTypes = {
+		submitForm: PropTypes.func.isRequired,
+
+		// from localize() HoC
+		translate: PropTypes.func.isRequired,
+	};
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			userType: '',
+		};
+	}
+
+	handleRadioChange = event => this.setState( { userType: event.currentTarget.value } );
+
+	handleSubmit = userType => {
+		//	const { userType } = this.state;
+
+		//	event.preventDefault();
+
+		this.props.recordTracksEvent( 'calypso_signup_actions_submit_user_type', {
+			value: userType,
+		} );
+
+		this.props.submitForm( userType );
+	};
+
+	getUserTypeQuestions = () => {
+		const { translate } = this.props;
+
+		return [
+			{
+				slug: 'creator',
+				label: translate( 'Myself' ),
+			},
+			{
+				slug: 'builder',
+				label: translate( 'Someone else' ),
+			},
+		];
+	};
+
+	renderUserTypeProfiles() {
```

Comment:
I don't see why we need a separate method for this - feels like an unnecessary indirection, especially given that the `render()` method is currently a 1-liner.

Reply:
Do you feel strongly about it? I think it is much clearer this way.

Reply:
I don't see how this is clearer 😉 Just a pure indirection, which makes it the opposite of being clear.